adapters/emx_digital/emx_digital.go

```diff
@@ -137,6 +137,41 @@ func buildImpBanner(imp *openrtb.Imp) error {
 	return nil
 }
 
+func buildImpVideo(imp *openrtb.Imp) error {
+	imp.Ext = nil
+
+	if len(imp.Video.MIMEs) == 0 {
+		return &errortypes.BadInput{
+			Message: fmt.Sprintf("Video: missing required field mimes"),
+		}
+	}
+
+	if imp.Video.H == 0 && imp.Video.W == 0 {
+		return &errortypes.BadInput{
+			Message: fmt.Sprintf("Video: Need at least one size to build request"),
+		}
+	}
+
+	if imp.Video.Protocols != nil {
+		imp.Video.Protocols = cleanProtocol(imp.Video.Protocols)
+	}
+
+	return nil
+}
+
+// not supporting VAST protocol 7 (VAST 4.0);
+func cleanProtocol(protocols []openrtb.Protocol) []openrtb.Protocol {
+	newitems := []openrtb.Protocol{}
```

Comment:
Consider using make to specify an initial capacity. It's slightly better memory management:
```
newitems := make([]openrtb.Protocol{}, 0, len(protocols))
```

Reply:
Makes sense to us, appreciate the suggestion